# Remove hardcoded CFLAGS
diff -urNad mysql-workbench-gpl-5.2.44-src-orig/ext/scintilla/gtk/Makefile.am mysql-workbench-gpl-5.2.44-src/ext/scintilla/gtk/Makefile.am
--- mysql-workbench-gpl-5.2.44-src-orig/ext/scintilla/gtk/Makefile.am	2012-09-26 23:50:57.000000000 +0300
+++ mysql-workbench-gpl-5.2.44-src/ext/scintilla/gtk/Makefile.am	2012-10-02 17:55:10.000000000 +0300
@@ -1,6 +1,6 @@
 AUTOMAKE_OPTIONS=no-dependencies
 
-AM_CXXFLAGS=-Wall -Wno-missing-braces -Wno-char-subscripts -pedantic -O0 -g3 -DGTK -DSCI_LEXER -DSCI_NAMESPACE
+AM_CXXFLAGS=-Wall -Wno-missing-braces -Wno-char-subscripts -pedantic -DGTK -DSCI_LEXER -DSCI_NAMESPACE
 
 #noinst_LTLIBRARIES=libwbscintilla.la
 pkglib_LTLIBRARIES=libwbscintilla.la
diff -urNad mysql-workbench-gpl-5.2.44-src-orig/ext/scintilla/gtk/Makefile.in mysql-workbench-gpl-5.2.44-src/ext/scintilla/gtk/Makefile.in
--- mysql-workbench-gpl-5.2.44-src-orig/ext/scintilla/gtk/Makefile.in	2012-09-26 23:51:06.000000000 +0300
+++ mysql-workbench-gpl-5.2.44-src/ext/scintilla/gtk/Makefile.in	2012-10-02 17:55:10.000000000 +0300
@@ -271,7 +271,7 @@
 top_srcdir = @top_srcdir@
 unixodbc = @unixodbc@
 AUTOMAKE_OPTIONS = no-dependencies
-AM_CXXFLAGS = -Wall -Wno-missing-braces -Wno-char-subscripts -pedantic -O0 -g3 -DGTK -DSCI_LEXER -DSCI_NAMESPACE
+AM_CXXFLAGS = -Wall -Wno-missing-braces -Wno-char-subscripts -pedantic -DGTK -DSCI_LEXER -DSCI_NAMESPACE
 
 #noinst_LTLIBRARIES=libwbscintilla.la
 pkglib_LTLIBRARIES = libwbscintilla.la
# Define absolute path for ifconfig command
diff -urNad mysql-workbench-gpl-5.2.44-src-orig/plugins/wb.admin/backend/wb_server_control.py mysql-workbench-gpl-5.2.44-src/plugins/wb.admin/backend/wb_server_control.py
--- mysql-workbench-gpl-5.2.44-src-orig/plugins/wb.admin/backend/wb_server_control.py	2012-09-26 23:50:56.000000000 +0300
+++ mysql-workbench-gpl-5.2.44-src/plugins/wb.admin/backend/wb_server_control.py	2012-10-02 17:55:10.000000000 +0300
@@ -32,7 +32,7 @@
 
 #-------------------------------------------------------------------------------
 def get_local_ip_list():
-    cmd    = "/bin/sh -c ifconfig"
+    cmd    = "/bin/sh -c /sbin/ifconfig"
     regexp = "inet addr:([0-9a-f:\.]+)"
 
     if hasattr(sys, 'getwindowsversion'):
